themes/default/content/blog/announcing-kubernetes-helm-release-resource-preview/index.md

```diff
@@ -0,0 +1,305 @@
+                          "metrics": {
+                              "enabled": True,
+                              "service": {
+                                  "annotations": {
+                                      "prometheus.io/port": "9127",
+                                  }
+                              },
+                          },
+                          "global": {
+                              "redis": {
+                                  "password": redis_password,
+                              }
+                          },
+                          "rbac": {
+                              "create": True,
+                          },
+                      },
+                  ),
+                  # By default Release resource will wait till all created resources
+                  # are available. Set this to true to skip waiting on resources being
+                  # available.
+                  skip_await=False)
+
+# We can lookup resources once the release is installed. The releases'
+# status field is set once the installation completes, so this, combined
+# with `skip_await=False` above, will wait to retrieve the Redis master
+# ClusterIP till all resources in the Chart are available.
+status = release.status
+srv = Service.get("redis-master-svc",
+                  Output.concat(status.namespace, "/", status.name, "-redis-master"))
+pulumi.export("redisMasterClusterIP", srv.spec.cluster_ip)
+
+```
+
+{{% /choosable %}}
+
+{{< /chooser >}}
+
+Users of the existing Helm Chart resource will notice that we retained the hierarchical `values` specification experience and a similar means to configure the repository. The above example also highlights the await and resource lookup experience.
+
+![helm-release](./helm-release.gif)
+
+## Which Helm resource to use and when?
+
+In summary, if your primary goal is to install Helm charts with configuration options natively supported by the chart, then Helm  Release should be a good fit. If on the other hand you intend to manipulate Helm Charts through transformations (due to bugs in the charts or restrictions in your environment), then the experience is much richer with the Helm Chart resource.
+
+If you intend to use Policy-as-Code to enforce resource policies at a Kubernetes resource granularity, then the Helm Chart component resource would be the preferred model to use.
+
+Note that both resources can co-exist safely in the same program.
+
+## What's next?
+
+The new Helm Release resource is available in *preview* now. During the preview the resource may evolve rapidly and users are requested to not use it for critical infrastructure. We have already received valuable feedback to shape this resource and we look forward to hearing more from our users. Please reach out by filing or commenting on issues on [Github](https://github.com/pulumi/pulumi-kubernetes/issues) and reaching out to us on [Community Slack](https://slack.pulumi.com). Happy charting!
```

Comment:
Nit: "reach out" is used twice in the last sentence

Reply:
Fixed.